frontend/src/pages/ActivityReport/Pages/goalsObjectives.js

```diff
@@ -116,11 +116,16 @@ const GoalsObjectives = ({
     if (index !== -1) {
       copyOfSelectedGoals.splice(index, 1);
     }
+
     onUpdateGoals(copyOfSelectedGoals);
 
     // if we have no goals, open the form up via the
     // hander provided by the context
     if (copyOfSelectedGoals.length === 0) {
+      setValue('goalForEditing', '');
+      setValue('goalName', '');
+      setValue('goalEndDate', '');
+      setValue('goalIsRttapa', '');
```

Comment:
Can we also add/update the test?

Reply:
https://github.com/HHS/Head-Start-TTADP/pull/1163/commits/7ea3c48c0280f4de424140bf7e2b855060dfa51d